config/jwt.js

```diff
@@ -0,0 +1,94 @@
+//Dependencies
+var bodyParser  = require('body-parser');
+var jwt    = require('jsonwebtoken');
+var key = require('./jwtkey');
+
+var mongoose = require('mongoose'),
+  User = mongoose.model('User');
+
+module.exports = function(app){
+  app.set('superSecret', key.secret);
+
+  // use body parser to get info from POST and/or URL parameters
+  app.use(bodyParser.urlencoded({ extended: false }));
+  app.use(bodyParser.json());
+
+  // API ROUTES -------------------
+
+  // route to authenticate a user and create token
+  // (POST ./api/auth/login) and (POST ./api/auth/signup)
+  var authToken = function(req, res) {
+    // find the user
+    User.findOne({
+      email: req.body.email
+    }, function(err, existingUser) {
+
+      if (err) throw err;
+
+      if (!existingUser) {
+        res.json({ success: false, message: 'User not found.',
```

Comment:
The purpose is that in `status()`, we send the necessary http status code alongside.